dockertest/subtest.py

```diff
@@ -154,27 +153,37 @@ def postprocess_iteration(self):
         """
         self.log_step_msg('postprocess_iteration')
 
-    @property
-    def control_config(self):
-        """
-        Represent operational control.ini's [Control] section as a dict or None
-        """
+    def _control_ini_section(self, section):
         if self._control_ini is None:
             fullpath = os.path.join(self.job.resultdir, 'control.ini')
             # Control-file & behavior cannot be assumed, file may not exist.
             try:
-                self._control_ini = config.ConfigDict('Control')
+                self._control_ini = config.ConfigDict(section)
```

Comment:
foo = self.control_config
    bz = self.bugzilla_config
    ! in the second invocation of _control_ini_section(), self._control_ini has been set (by the first invocation) to config.ConfigDict('control')